sqlsynthgen/main.py

```diff
@@ -44,10 +58,14 @@ def create_tables(orm_file: str = typer.Argument(...)) -> None:
 
 
 @app.command()
-def make_generators(orm_file: str = typer.Argument(...)) -> None:
+def make_generators(
+    orm_file: str = typer.Argument(...),
+    config_file: Optional[str] = typer.Argument(None),
+) -> None:
     """Make a SQLSynthGen file of generator classes."""
     orm_module = import_file(orm_file)
-    result = make_generators_from_tables(orm_module)
+    generator_config = read_yaml_file(config_file)
```

Comment:
Further to the above, we could move the config path check here

```suggestion
    generator_config = read_yaml_file(config_file) if config_file else {}
```

Reply:
Done